tests/framework/utils.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Generic utility functions that are used in the framework."""
+import subprocess
```

Comment:
nit: order importa alphabetically

Reply:
Will fix, thanks.

I think that these checks should be added in the linter. It should be doable to do it through one of the current linters.